src/System.IO.Pipes/src/System/IO/Pipes/AnonymousPipeServerStream.Windows.cs

```diff
@@ -16,6 +17,12 @@ public sealed partial class AnonymousPipeServerStream : PipeStream
     {
         // Creates the anonymous pipe.
         private void Create(PipeDirection direction, HandleInheritability inheritability, int bufferSize)
+        {
+            Create(direction, inheritability, bufferSize, null);
+        }
+
+        // Creates the anonymous pipe.
+        private void Create(PipeDirection direction, HandleInheritability inheritability, int bufferSize, PipeSecurity pipeSecurity)
```

Comment:
Add comment for the parameter `pipeSecurity` to indicate that it is used by public constructors in Mono.